### ISSUE TITLE
Not able anymore to download video files

### ISSUE THREAD

**Issue by piotrv** [przemyslawpluta/mongo-edu#72]
```
npm list -g --depth=0
/usr/lib
├── mongo-edu@0.2.53
```
`> mongo-edu -d video/ -u mail@me.com --cw -co --hq --save video`

> ? Select Courses: Current ? Hi PIOTREK 😄 Found 1 Course. Select: M102: MongoDB for DBAs i Unable to locate any videos lists in the wiki. Are videos list present?

`> mongo-edu -d video/ -u mail@me.com --cwd -co --hq --save video`

> i Unable to locate any videos lists in the wiki. Are videos list present? ? Default wiki search returned no resuts. Perform courseware search with --cw ?
> Yes i Looks like the course is not yet available or has already ended. Videos list is not available. Check the start/end date for selected course. 

Used to work with previous courses I attended...

**Comment by przemyslawpluta** [comment]
@piotrv can't replicated just tested against latest version on `*nix`, `mac os*` and `win*` and I'm getting all the courses without any issues.

**Comment by piotrv** [comment]
Thanks for quick response. 
This is weird. I do get the handout files btw ...
If you cannot replicate the issue, I can only think of a user account related issue.
I must check again with  another created account...

`node -v`
> v4.4.0

`python -V`
> Python 2.7.6

` cat /etc/os-release `
> NAME="Ubuntu"
> VERSION="14.04.4 LTS, Trusty Tahr"

**Comment by piotrv** [comment]
I created a new 'test' account at university.mongodb.com applying the related test email address for the  _-u_ flag of mongo-edu: now I can download all video files.
This is a quick workaround, but is not a solution since I want to use my current account.

So as it seems , the issue is related to my account record  in the mongodb database.
But nothing has been changed in my profile.
If this is not a bug in mongo-edu, this issue should still be addressed since it is out of my control...
